lib/sensor.js

```diff
@@ -206,7 +205,9 @@ class Sensor extends Withinable {
             }
 
             mapped = Fn.fmap(raw, this.range[0], this.range[1], state.scale[0], state.scale[1]);
-            constrain = Fn.constrain(mapped, state.scale[0], state.scale[1]);
+            // Code.org: round scaled sensor values
```

Comment:
Just curious - do we have a standard way of tagging Code.org specific updates in this repo? (is it just adding a comment with `// Code.org:`?)

Reply:
Looking at prior PRs that added Code.org customizations, @islemaster seemed to consistently use this convention, as seen [here](https://github.com/code-dot-org/johnny-five/pull/9/files).